.github/auto_assign.yml

```diff
@@ -7,7 +7,7 @@ addAssignees: true
 # A list of reviewers to be added to pull requests (GitHub user name)
 reviewers: 
   - azheregelya
```

Comment:
Could you please remove azheregelya from the list of reviewers? The request for this was already raised sometime back but it was missed I think.

Reply:
done